lms/djangoapps/course_api/blocks/transformers/tests/test_block_completion.py

```diff
@@ -49,7 +49,10 @@ class BlockCompletionTransformerTestCase(TransformerRegistryTestMixin, ModuleSto
     def setUp(self):
         super(BlockCompletionTransformerTestCase, self).setUp()
         self.user = UserFactory.create(password='test')
-        self.override_waffle_switch(True)
+
+        _waffle_overrider = completion_waffle.waffle().override(completion_waffle.ENABLE_COMPLETION_TRACKING, True)
+        _waffle_overrider.__enter__()
```

Comment:
Why do we have to call the member function this way to make override work?

Reply:
I think I can replace that with a Mixin now.